fedora/fas.py

```diff
@@ -128,19 +116,7 @@ async def hello(self, evt: MessageEvent, username: str) -> None:
             message += " - " + " or ".join(pronouns)
         await evt.respond(message)
 
-    @command.new(help="Return brief information about a Fedora user.", aliases=ALIASES["user"])
-    @command.argument("username", pass_raw=True, required=False)
-    async def user(self, evt: MessageEvent, username: str) -> None:
-        """
-        Returns a information from Fedora Accounts about the user
-        If no username is provided, defaults to the sender of the message.
-
-        #### Arguments ####
-
-        * `username`: A Fedora Accounts username or a Matrix User ID
-           (e.g. @username:fedora.im)
-
-        """
+    async def _user_info(self, evt: MessageEvent, username: str) -> None:
```

Comment:
If `username` can be `None`, let's state it in the type hints: `username: str | None`